config/routes.rb

```diff
@@ -3300,32 +3299,30 @@
   controller_routes.each do |controller_name, controller_actions|
     # Default route with no action to controller's index action
     unless routes_without_index.include?(controller_name)
-      match controller_name.to_s, :controller => controller_name, :action => :index, :via => :get
+      get controller_name.to_s, :controller => controller_name, :action => :index
     end
 
     default_routes.each do |action_name|
       post "#{controller_name}/#{action_name}(/:id)",
+           :action     => action_name,
+           :controller => controller_name
+    end
+
+    # One-by-one get/post routes for defined controllers
+    next unless controller_actions.kind_of?(Hash)
```

Comment:
This was intentionally an if condition previously because we wanted to avoid using guard clauses since this method is very long and someone could add code to the end not realizing it would bail out here.  Let's undo the rubocops for guard clause conversion, and perhaps add rubocop:disable comments

Reply:
fixed